webapp/src/components/Ferdous.svelte

```diff
@@ -0,0 +1,12 @@
+<script lang="ts">
+  import { Chart, LineSeries } from 'svelte-lightweight-charts';
+  export let graphData: { time: string, value: number }[] = [];
+  export let width: number = 800;
+  export let height: number = 600;
+</script>
+
+<div>
+  <Chart {width} {height}  >
+    <LineSeries data={graphData} reactive={true}/>
+  </Chart>
+</div>
```

Comment:
This is a little nit picky but always add an extra newline at the end of every file so git does not give this red mark